test/utility/MathEx.ts

```diff
@@ -16,7 +16,7 @@ describe('MathEx', () => {
         mathContract = await Contracts.TestMathEx.deploy();
     });
 
-    const testMulDiv = (x: BigNumber, y: BigNumber, z: BigNumber) => {
+    const test = (x: BigNumber, y: BigNumber, z: BigNumber) => {
```

Comment:
test what? :)

Reply:
Well, it now tests the function `mulDivF`, `mulDivC` and `minFactor`.
So no sure I have a better name for it.
Any suggestions?

Reply:
testMulDivAndMinFactor

Reply:
Fixed